Makefile

```diff
@@ -239,11 +70,23 @@ build-all: $(addprefix sub-build-,$(VALIDARCHES))
 sub-build-%:
 	$(MAKE) build ARCH=$*
 
+# Build mounts for running in "local build" mode. This allows an easy build using local development code,
+# assuming that there is a local checkout of libcalico in the same directory as this repo.
+PHONY:local_build
```

Comment:
Should this be inside the `ifdef` below?

Reply:
We need the `local_build` rule for when we _don't_ run a local build, so the PHONY label needs to be there.

Reply:
Compare with the [change I made to the felix build](https://github.com/projectcalico/felix/commit/27a087ca019db79a12849aa06843c5b71f317c2b#diff-b67911656ef5d18c4ae36cb6741b7965) to change the local_build dependency into a variable.  

Depending on a phony is a big smell in a makefile, it always forces a rebuild of the target.  Removing that took the FV cycle time down from 50s to 20s, which is a big win.

Reply:
FIxed